webapp/sources/rudder/rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -137,7 +137,7 @@ rudder.jdbc.batch.max.size=500
 # Defaults: archive disabled, delete after 4 days.
 #
 
-rudder.batch.reportscleaner.archive.TTL=4
+rudder.batch.reportscleaner.archive.TTL=0
```

Comment:
I think we should keep that one, so that if someone set "archive reports", it doesn't blow up too much

Reply:
OK, so no: it's the time for archiving, so we really want it to be 0 to disable archiving.